dashboard/app/controllers/home_controller.rb

```diff
@@ -34,6 +34,23 @@ def health_check
 
   GALLERY_PER_PAGE = 5
   def index
+    if request.cookies['pm'] == 'new_header'
```

Comment:
I verified in `irb` that even if the hash doesn't have a `'pm'` entry, then the two queries work okay.  At this point I'm assuming that requests will have a `cookies` object, even if empty.